### DISCUSSION TITLE
Store array items in a different rows

### DISCUSSION THREAD

**Discussion by JoshiShivraj** [Budibase/budibase#8873] [Help]
Hi Folks,

I am recently came across budibase and I am developing one app where I have to take multiple system for building like lets say I have one building called : ABC and this building has different system like AC, Heater etc.

Now what I is happening is when I am selecting building ABC and lets say I am selecting AC and Heater with muliselect picker then 
table is storing data like this format  
Building name        System
ABC                        AC,Heater

but I am expecting is it should store like this:
Building name        System
ABC                        AC
ABC                        Heater

I have tried javascript for looping but it didn't work, can you guys please help me its urgent.

Thanks In advance

**Comment by melohagan** [comment]
Hey @JoshiShivraj 

Could you provide some screenshots please? I'm not quite sure what your issue is.
